base/applications/cmdutils/where/lang/en-US.rc

```diff
@@ -0,0 +1,52 @@
+LANGUAGE LANG_ENGLISH, SUBLANG_ENGLISH_US
+
+STRINGTABLE
+BEGIN
+    IDS_USAGE "Usage: WHERE [options] pattern...\n\
```

Comment:
So, "WHERE"s in the help message should be lowercase?

Reply:
No personal preference about that; how is it done in the other tools in cmdutils?

Reply:
Case in `cmdutils` is inconsistent but UPPERCASE is often used.

https://github.com/reactos/reactos/blob/master/base/applications/cmdutils/attrib/lang/en-US.rc#L7
https://github.com/reactos/reactos/blob/master/base/applications/cmdutils/taskkill/lang/en-US.rc#L5
https://github.com/reactos/reactos/blob/master/base/applications/cmdutils/xcopy/lang/en-US.rc#L25

Reply:
`Usage: lowercase ...` in Linux-like system.
`UPPERCASE ...` in Windows.

Reply:
Yes we tend to use uppercase since we are making a re-creation of Windows, and not of *nix-like OS.